### ISSUE TITLE
VS  doesn't refresh to the correct repo after creating a new one

### ISSUE THREAD

**Issue by amytruong** [github/VisualStudio#71]
VS doesn't refresh to the correct repo after creating a new one

**Steps to reproduce:**
1. Have an existing project open
2. Navigate to Team Explorer --> Manage Connections 
3. Click Create 
4. Enter in required fields and click Create to create a new repo

**Expected Results:** VS switches to the newly created repo/project

**Actual Results:**  VS stays on the previously opened repo/project and you have to click on the newly created project/repo in the Local Git Repositories list in Team Explorer to get to it. 

![create](https://cloud.githubusercontent.com/assets/10725993/9690987/7b6e83b4-52f4-11e5-9d62-c48142ea7cde.gif)

**Comment by shana** [comment]
Yup, this is a problem. We don't have access to APIs that would allow us to switch to the new repo automatically, so we end up a bit stuck.

/cc @clboles @mtbandolier @acangialosi

**Comment by amytruong** [comment]
@shana gotcha! This seems like something I can close out then?

**Comment by shana** [comment]
@amytruong Nope, leave it open, it's a good way to track things that need new VS apis.
